src/collapse/perf.rs

```diff
@@ -5,11 +5,13 @@ use std::fs::File;
 use std::io;
 use std::io::prelude::*;
 
+use crate::collapse::Frontend;
+
 const TIDY_GENERIC: bool = true;
 const TIDY_JAVA: bool = true;
 
-#[derive(Debug, Default)]
-pub struct Options {
+#[derive(Clone, Debug, Default)]
+pub struct PerfOptions {
```

Comment:
I think this should continue to just be named `Options` since it'll only ever be accessed through the `perf` module.

Reply:
👍 - With the module change, makes sense!